src/main/resources/templates/product/show_products.ftl

```diff
@@ -90,10 +90,6 @@
                         <button class="btn btn-primary" type="submit">Activate</button>
                         <input type="hidden" name="${_csrf.parameterName}" value="${_csrf.token}"/>
                     </form>
```

Comment:
Please, also set appropriate `<title>`s for pages in scope of this PR.
For example `<title>Products</title>` for products and so on.